spec/controllers/state_file/questions/return_status_controller_spec.rb

```diff
@@ -2,6 +2,24 @@
 
 RSpec.describe StateFile::Questions::ReturnStatusController do
   describe "#edit" do
+    context "assignment of various instance variables" do
+      it "assigns them correctly" do
+        az_intake = create(:state_file_az_intake)
+        sign_in az_intake
+        create(:efile_submission, :notified_of_rejection, :for_state, data_source: az_intake)
+        get :edit, params: { us_state: "az" }
+
+        expect(assigns(:refund_url)).to eq "https://aztaxes.gov/home/checkrefund"
+        expect(assigns(:tax_payment_url)).to eq "AZTaxes.gov"
+        expect(assigns(:download_form_name)).to eq "Form AZ-140V"
+        expect(assigns(:mail_voucher_address)).to eq "Arizona Department of Revenue<br/>"\
+          "PO Box 29085<br/>"\
+          "Phoenix, AZ 85038-9085"
+        expect(assigns(:voucher_path)).to eq "/pdfs/AZ-140V.pdf"
+        expect(assigns(:survey_link)).to eq "https://codeforamerica.co1.qualtrics.com/jfe/form/SV_7UTycCvS3UEokey"
+      end
+    end
```

Comment:
2 things:
1. i wonder if this test is overkill? actually what might be better is to have a feature spec that looks for these attributes in the view but adding that as part of this PR seemed a little extra
2. i don't love having all these strings in here but it would be annoying to mock the StateInformationService for every single one...

Reply:
I agree with extracting attributes from the view, but I do actually like that the strings are explicitly stated in the text and would not mock that - having them here is simple and explicit - exactly what we want from unit tests.